.buildkite/pipeline.yml

```diff
@@ -32,20 +59,18 @@ steps:
         account_ids: "436617320021"
         no-include-email: true
         region: "us-west-2"
-      docker#v2.1.0:
+      docker#v3.1.0:
+        shell: ["/bin/bash", "-i", "-e", "-c"]
         debug: true
-        image: "436617320021.dkr.ecr.us-west-2.amazonaws.com/ci:amazonlinux2_2-3"
+        image: "436617320021.dkr.ecr.us-west-2.amazonaws.com/ci:amazonlinux2_2-4"
```

Comment:
We should be using labels on docker images rather than changing the image names.    
Ex:    
`image: "436617320021.dkr.ecr.us-west-2.amazonaws.com/ci:amazonlinux2_2:latest"`